yaas/cmd.py

```diff
@@ -12,18 +20,22 @@ def version(parser, args):
     """print the yaas version"""
     print YAAS_VERSION
 
-def hello(parser, args):
-    """show an elephant"""
-    print "                .-.._       "
-    print "          __  /`     '.     "
-    print "       .-'  `/   (   a \    "
-    print "      /      (    \,_   \   "
-    print "     /|       '---` |\ =|   "
-    print "    ` \    /__.-/  /  | |   "
-    print "       |  / / \ \  \   \_\  "
-    print "       |__|_|  |_|__\       "
+def _response_hook(res, *args, **kwargs):
+    print "Request:", res.request.__dict__
+    print "Response:", res.__dict__
 
 def main():
+    if 'YAAS_SERVER_URL' in os.environ:
+        config.server_url = os.environ['YAAS_SERVER_URL']
+
+    user = config.requests_kwargs['auth'][0]
+    password = config.requests_kwargs['auth'][1]
+    if 'YAAS_USER' in os.environ:
+      user = os.environ['YAAS_USER']
+    if 'YAAS_PASSWORD' in os.environ:
+      password = os.environ['YAAS_PASSWORD']
+    config.requests_kwargs['auth'] = (user, password)
```

Comment:
IMO, `config.requests_kwargs['user']` is more readable than `config.requests_kwargs['auth'][0]`. Are the benefits I'm not seeing to using a tuple?